components/kserve/kserve.go

```diff
@@ -22,6 +24,11 @@ var imageParamMap = map[string]string{
 	"kserve-storage-initializer": "RELATED_IMAGE_ODH_KSERVE_STORAGE_INITIALIZER_IMAGE",
 }
 
+var dependentImageParamMap = map[string]string{
+	"odh-model-controller": "RELATED_IMAGE_ODH_MODEL_CONTROLLER_IMAGE",
+	"kserve-enabled":       "true",
```

Comment:
Is this parameter used by the manifests?

Reply:
Yes. These 2 parameters are required for the manifests

Reply:
The variable name implies only mapping for image parameters, but this also includes a boolean flag to enable functionality within the component. Let me know if I should rename the map

Reply:
Correct, but since its not an environment variable, the value will be static. Not sure if this needs to be added to the map given that you already set it here in manifests https://github.com/opendatahub-io/odh-manifests/pull/912/files#diff-c07700bf666d03f737fbfc6e4f1ab82eb46d1e73aa4d038e1d953001b993169dR2

Reply:
Does the `params.env` file get applied with the manifests? I thought it got converted into the imagemap and then got applied. 
Can you confirm if having the flag set in `params.env` in odh-manifests is enough to apply the flag? If yes I can remove it from the `imageMap` here

Reply:
Yes, its enough to apply the flag in `params.env`. The imagemap is just a mapping when you want to override existing defaults

Reply:
Done